src/lib/parquet/src/Flow/Parquet/ParquetFile/Schema.php

```diff
@@ -19,12 +18,20 @@ final class Schema
     /**
      * @param array<Column> $columns
      */
-    public function __construct(
+    private function __construct(
         private readonly FlatColumn $schemaRoot,
         private readonly array $columns
     ) {
     }
 
+    public static function from(Column ...$columns) : self
+    {
+        return new self(
+            FlatColumn::boolean('schema'),
```

Comment:
Can we have phpdoc why this is set here? From the name, it's not clear that there is such a setting

Reply:
it was just the first step before #622